src/controllers/dynakube/istio/istio.go

```diff
@@ -19,41 +14,21 @@ var (
 	IstioGVR        = fmt.Sprintf("%s/%s", IstioGVRName, IstioGVRVersion)
 )
 
-// BuildNameForEndpoint returns a name to be used as a base to identify Istio objects.
-func BuildNameForEndpoint(name string, commHosts []dtclient.CommunicationHost) string {
-	result := make([]string, len(commHosts))
-	for index, commHost := range commHosts {
-		result[index] = fmt.Sprintf("%s-%s-%s-%d", name, commHost.Protocol, commHost.Host, commHost.Port)
-	}
-	sum := sha256.Sum256([]byte(strings.Join(result, "\n")))
-	return hex.EncodeToString(sum[:])
-}
-
+// TODO: Maybe move to Client
 // CheckIstioInstalled run discovery query for server resource for group version
-func CheckIstioInstalled(discoveryclient discovery.DiscoveryInterface) (bool, error) {
-	_, err := discoveryclient.ServerResourcesForGroupVersion(IstioGVR)
+func CheckIstioInstalled(discoveryClient discovery.DiscoveryInterface) (bool, error) {
```

Comment:
I think it's a good idea to move it to a new istio client. Should be easy refactoring, since we use it only in 2 places:
- https://github.com/dynatrace/dynatrace-operator/blob/11d19c8cc31d8ae1bf12d771a3114ccc6ab9ad56/src/webhook/validation/dynakube/istio.go#L21
and in `setupIstio`. And we always have instance of istio client :)

Reply:
fixed